### DISCUSSION TITLE
`StoryObj` possibly not recognizing Radix Primitive component type

### DISCUSSION THREAD

**Discussion by TylerAPfledderer** [storybookjs/storybook#28477] [Help]
### Summary

Hello!

What's the recommended approach to working with Radix primitives in Storybook with Typescript?

Using `satisfies Meta<typeof Accordion.Root>` and  `StoryObj<typeof meta>` is providing this initial type error for a story:
```bash
Type '{ render: () => JSX.Element; }' is not assignable to type 'StoryAnnotations<ReactRenderer, never, never>'.
  Property 'args' is missing in type '{ render: () => JSX.Element; }' but required in type '{ args: never; }'
```
args `never`?

Seems like `StoryObj` does not recognize the complex type from `Accordion.Root` as a component type.
```ts
ForwardRefExoticComponent<(AccordionSingleProps | AccordionMultipleProps) & RefAttributes<HTMLDivElement>>
```

**Comment by Dalcvi** [comment]
Just in case, one of the solutions that worked for me was to give the component in StoryObj instead of giving typeof meta:

`type Story = StoryObj<typeof AccordionRowComp>;`
